start-training.sh

```diff
@@ -9,3 +9,16 @@ export ROBOMAKER_COMMAND="./run.sh run distributed_training.launch"
 export CURRENT_UID=$(id -u):$(id -g)
 
 docker-compose -f ./docker-compose.yml up -d
+
+#sleep for 20 seconds to allow the containers to start
+sleep 20
+
+echo 'Attempting to pull up sagemaker logs...'
+SAGEM_ID="$(docker ps | awk ' /sagemaker/ { print $1 }')"
+gnome-terminal -x sh -c "docker logs -f $SAGEM_ID"
```

Comment:
This might help

```
command -v gnome-terminal &>/dev/null
if [[ $? -eq 0 ]]; then
    gnome-terminal -x sh -c "docker logs -f $SAGEM_ID"
fi
```

Reply:
ok, makes sense, let me fix it